.github/workflows/docker-publish-full.yml

```
@@ -21,87 +22,128 @@ concurrency:
   cancel-in-progress: true
   group: ${{ github.workflow }}-${{ github.ref }}
 
+# The name for the produced image at ghcr.io.
 env:
-  IMAGE_NAME: wetterdienst-full
+  GHCR_IMAGE_NAME: wetterdienst-full
```

I think the full name has to go here.

```suggestion
  GHCR_IMAGE_NAME: earthobservations/wetterdienst-full
```